environment.rb

```diff
@@ -0,0 +1,139 @@
+module Environment
+    class Depot
+        attr_accessor :packs
+        def initialize 
+            @packs = {
+                :simple_transportation_pack => {
+                    :intelligence => "cellphone",
+                    :items => "mediapack, chevy_versa"
+                },
+                :standard_transportation_pack =>{
+                    :intelligence => "cellphone, antena",
+                    :arsenal => "colt_1911",
+                    :items => "handcuff, mediapack, chemistry, financial"
+                },
+                :simple_mission_pack =>{
+                    :intelligence => "infopack, laptop, cellphone, antenna",
+                    :arsenal => "colt_191",
+                    :items => "handcuffs, mediapack, chemistry, financial"
+                },
+                :standard_mission_pack =>{
+                    :intelligence => "infopack, laptop, cellphone, antenna",
+                    :arsenal => "remington_870, colt_1911, machete, hatchet",
+                    :items => "handcuffs, mediapack, chemistry, financial"
+                }
+            }
+        end
+        
```

Comment:
Try to avoid extra empty lines at the end of the methods, they are taken as a bad practice. I would delete this one.